src/spatial_grid.jl

```diff
@@ -1,35 +1,21 @@
 """
-` Rasterize a points cloud in 2D`
+    rasterize_points(cloud::PointCloud, dx::AbstractFloat)
+    rasterize_points(cloud::Matrix{AbstractFloat}, dx::AbstractFloat)
```

Comment:
Mmm... yes that is vaguely familiar. But for now `rasterize_points{T1 <: AbstractFloat, T2 <: AbstractFloat}(cloud::Matrix{T1}, dx::T2)` will suffice.